ibis/__init__.py

```diff
@@ -40,6 +40,11 @@
     pass
 
```

Comment:
need a small section in ``setup.py`` to add additional requires, something like: https://github.com/ibis-project/ibis/pull/1167/files#diff-2eeaed663bd0d25b7e608891384b7298

Reply:
addressed in [DOC: add bigquery_requires to setup.py ](https://github.com/ibis-project/ibis/pull/1170/commits/cb16e5bec119a2d41d2f8784b93865b9679a6f6b)